helpers/test/helpers.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
+The complete set of authors may be found at http://polymer.github.io/AUTHORS
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
+-->
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>test for app-header</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+
+  <script src="../../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../../web-component-tester/browser.js"></script>
+  <script src="../../../test-fixture/test-fixture-mocha.js"></script>
+  <link rel="import" href="../../../test-fixture/test-fixture.html">
+  <link rel="import" href="../helpers.html">
+
+  <style>
+    #region {
+      height: 500px;
+      overflow: hidden;
+    }
+
+    .content {
+      height: 10000px;
+      width: 10000px;
+    }
+  </style>
+</head>
+<body>
+
+  <div id="region">
+    <div class="content"></div>
+  </div>
+  <div class="content"></div>
+
+  <script>
+
+    suite('helper functions', function() {
+
+      test('Polymer.AppLayout.scrollTo', function(done) {
+        var x = 500;
+        var y = 500;
+        var region = document.querySelector('#region');
+
+        Polymer.AppLayout.scrollTo(x, y);
+
+        Polymer.Base.async(function() {
+          assert.equal(window.pageXOffset, x, 'document scrollLeft');
+          assert.equal(window.pageYOffset, y, 'document scrollTop');
+
+          Polymer.AppLayout.scrollTo(x, y, region);
+
+          Polymer.Base.async(function() {
+            assert.equal(region.scrollLeft, x, 'region scrollLeft');
+            assert.equal(region.scrollTop, y, 'region scrollTop');
+            done();
+          }, 100);
+
+        }, 100);
```

Comment:
Could make this two tests (one for doc scroll, one for element scroll) to avoid nesting async's.